pre_commit/error_handler.py

```diff
@@ -29,6 +31,9 @@ def _log_and_exit(msg, exc, formatted):
         five.to_bytes(msg), b': ',
         five.to_bytes(type(exc).__name__), b': ',
         _to_bytes(exc), b'\n',
+        _to_bytes('pre-commit.version={}\n'.format(pre_commit_version)),
+        _to_bytes('sys.version={}\n'.format(sys.version.replace('\n', ' '))),
+        _to_bytes('sys.executable={}\n'.format(sys.executable)),
     ))
     output.write(error_msg)
```

Comment:
the version info should only go to the log file, right now it's being printed to the output as well

Reply:
Can you please explain the motivation behind this? My thought were that such information is useful when creating an issue/debugging custom hooks, so copy-paste from console would be a common use case

Reply:
the error output is supposed to be intentionally brief (don't show debugging info / stacktraces to users) with the actual meat of the bug-reportable information being in the error log file